root/static/scripts/edit/URLCleanup.js

```diff
@@ -3232,7 +3232,8 @@ const CLEANUPS: CleanupEntries = {
     match: [new RegExp('^(https?://)?(www\\.)?jazzmusicarchives\\.com', 'i')],
     restrict: [LINK_TYPES.otherdatabases],
     clean: function (url) {
-      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?jazzmusicarchives\.com\/([^#]+)(?:#.*)?$/, 'https://www.jazzmusicarchives.com/$1');
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?jazzmusicarchives\.com\/([^#?]+)(?:[#?].*)?$/, 'https://www.jazzmusicarchives.com/$1');
+      url = url.replace(/^(https:\/\/www\.jazzmusicarchives\.com\/(\w+)\/(?:[\w%()-]+\/)?[\w%()-]*)\/$/, '$1');
```

Comment:
What is the purpose of this line? Removing the trailing slash?

Reply:
Yes, if you have a good way to drop the (optional) trailing slash in the previous line, that'd be fine as well of course :)

Reply:
Did not test: 
```suggestion
      url = url.replace(/\/+$/, '');
```